deployment/terraform/create.go

```diff
@@ -14,11 +14,9 @@ import (
 	"os/exec"
 	"strings"
 	"time"
-	"unicode"
-	"unicode/utf8"
 
-	"github.com/mattermost/mattermost-load-test-ng/loadtest"
-	"github.com/mattermost/mattermost-load-test-ng/terraform/ssh"
+	"github.com/mattermost/mattermost-load-test-ng/deployment"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
```

Comment:
Couldn't `ssh` be used from something else than terraform? I am wondering what would be the best place for that to reside. Not proposing anything, just interested in your thoughts.

Reply:
Yes totally. That was a conscious choice. I am refraining from prematurely optimizing package design until we get a clear picture of how deployments will look in the future. This will likely move one-level up under deployment. But maybe there are more deployers ? Maybe we get rid of the ssh package ? Until then, let's keep it under terraform.

Reply:
Yes, we'll figure it out. All good for now :+1: